### PR TITLE
Add book

### PR DESCRIPTION
**Pull request by qwibbler**
- Add store
- Add actions: ADD_BOOK, REMOVE_BOOK
- Implement add/remove book
- Use useSelector to subscribe to store and display current books.

### CODE REVIEW

File: src/components/add-book.js [hunk 1]
```diff
@@ -1,10 +1,33 @@
...
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <form>
-    <input type="text" placeholder="Book Title" />
-    <input type="select" placeholder="Category" />
-    <input type="button" value="Add Book" />
-  </form>
-);
+const AddBook = () => {
```
Comment (omar-labana) [inline src/components/add-book.js:6]: - [x] The form responsible for adding a book should be a controlled component. The component should contain: - A local state with `title`, `author` - they should have their inputs that updates the local state every time they update (`onChange`) then dispatches the `addBook` action with the object containing a new id as its argument

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @qwibbler, 

Excellent work so far! 👌
You have implemented all the required features correctly and without any errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/add-book.js (hunk 1 above)
Comment (qwibbler) [reply src/components/add-book.js:6]: You mean, have someting like:

```
const [title, setTitle] = useState('');
const [author, setAuthor] = useState('Anonymous');
const handleChange = () => {
  change => setTitle, setAuthor
}

const submitBookToStore = () => {
  if (title.value) {
    const newBook = {
      id: uuidv4(),
      title,
      author,
    }
    dispatch(newBook)
  }; 
}

button.onclick = submitBookToStore
```

???

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Zaman, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/add-book.js (hunk 1 above)
Comment (rloterh) [reply src/components/add-book.js:6]: This issue is well resolved! ✔️

**Pull request merged**